docs/components/cc-explore.md

```diff
@@ -0,0 +1,114 @@
+## CC Explore Banner
+This is an expandable component which contains links to other CC Web Properties. This component renders as a button on the top-right corner of the page. It should be placed above other components (navigation etc) so that it maintains this position. When this button is clicked, the component is expanded to reveal the different links.
+
+---
+### Attributes
+#### Donation URL
+This component accepts one attribute `donation-url` which is used for the Donation button contained in the component. This URL is what is redirected to when the donation button is clicked. 
+
+### Usage
+To use this component, import the CDN version of the CC Global components libary and initialize it. 
+Then render it as:
+
+```html
+<cc-explore
+    donation-url="http://example.com"
+/>
+```
+
+This will render as shown below (the black button on the right).
+
+---
+
+<cc-explore />
+
+### Code
+```vue
+<template>
```

Comment:
Let's not manually duplicate the code here since it will quickly go out-of-sync from the actual source code.